pkg/signing/signing.go

```diff
@@ -358,10 +348,10 @@ func Verify(cpath, url string, id uint32, isGroup bool, authToken string, noProm
 			name = i.Name
 			break
 		}
-		authok += fmt.Sprintf("\t%s, KeyID %X\n", name, signer.PrimaryKey.KeyId)
+		author += fmt.Sprintf("\t%s, KeyID %X\n", name, signer.PrimaryKey.KeyId)
 	}
 	fmt.Printf("Data integrity checked, authentic and signed by:\n")
-	fmt.Print(authok)
+	fmt.Print(author)
```

Comment:
Let's move this to previous line.

`fmt.Printf("Data integrity checked, authentic and signed by:\n%v", author)`